vllm/utils.py

```diff
@@ -1135,3 +1135,29 @@ async def _run_task_with_lock(task: Callable, lock: asyncio.Lock, *args,
     """Utility function to run async task in a lock"""
     async with lock:
         return await task(*args, **kwargs)
+
+
+def load_general_plugins():
```

Comment:
I think we should move this to a dedicated `vllm.plugins` module so it can be easily found by plugin developers who want to know what is going on under the hood.

Reply:
makes sense, added in [bd828b7](https://github.com/vllm-project/vllm/pull/7426/commits/bd828b76798d8217b7ee5fd46428c44d9d9d1927)